web/client/actions/security.js

```diff
@@ -168,3 +173,59 @@ export function verifySession() {
 }
 
 export const checkLoggedUser = () => ({type: CHECK_LOGGED_USER});
+
+/**
+ * set status to show modal for inserting credentials
+ * @param {boolean} status
+ */
+export const setShowModalStatus = (status) => {
+    return {
+        status,
+        type: SET_SHOW_MODAL_STATUS
+    };
+};
+/**
+ * set protected services to request to provide username and password
+ * @param {object[]} protectedServices
+ */
+export const setProtectedServices = (protectedServices) => {
+    return {
+        protectedServices,
+        type: SET_PROTECTED_SERVICES
+    };
+};
+/**
+ * clear security of layers
+ * @param {string} id
+ */
+export const clearSecurity = (protectedId) => {
```

Comment:
Probably this should be called clearSecurityLayers if the only purpose is to clear the layers security